homeassistant/components/asuswrt/sensor.py

```diff
@@ -99,6 +108,12 @@ class AsuswrtTXSensor(AsuswrtSensor):
 
     _name = "Asuswrt Upload Speed"
     _unit = DATA_RATE_MEGABITS_PER_SECOND
+    _icon = _UPLOAD_ICON
+
+    @property
+    def icon(self):
+        """Return the icon."""
+        return self._icon
```

Comment:
Just return it directly, no need to assign it as a class instance variable. (Applies to other as well).

```suggestion
    @property
    def icon(self):
        """Return the icon."""
        return UPLOAD_ICON
```

Reply:
Totally agree :) I was surprised with _unit style but followed it